### ISSUE TITLE
Learner view of course resource is missing proper title and main heading

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#14522]
<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

❌ **This issue is not open for contribution. Visit <a href="https://learningequality.org/contributing-to-our-open-code-base/" target="_blank">Contributing guidelines</a>** to learn about the contributing process and how to find suitable issues.

<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

**Target branch:** release-0.19.x, tested specifically the asset from #14452 

## Observed behavior
When learner is navigating and progressing through the resources in the course, it can be very disorienting not to have a clear idea which resource is currently in the view, as all of them are missing the main heading, and something more specific in the browser title (this is a general issue, not specifically for courses). Even with the sidebar opened (@pcenov mentioned that we may consider keeping the sidebar opened by default when the viewport view width allows it), the visual aspect of the **current** resource in the sidebar is not _grounding_ enough, as it serves mostly to indicate where the _current_ resource is respect to all the rest of them in the Unit, and not as a **title** for what activity is the learner involved in at that time.  

The only thing that sort of _looks like_ a title/heading is the course title in the top bar, but it is not presented as such semantically, and just the course title is **not enough** information to allow the learner to know where are they and what are they currently working on. In the case of the exercise/practice resources, the **Get N correct** can be even more confusing, as it's definitely not the title of the resource.

Reading | Practice
-- | --
<img width="1186" height="1146" alt="Image" src="https://github.com/user-attachments/assets/b47cb13a-c3c9-4c0d-9fce-51bf428308e6" /> | <img width="1186" height="1146" alt="Image" src="https://github.com/user-attachments/assets/ce323fe5-91ad-47da-bd52-7e70b00f8e7f" />

## Errors and logs
…

## Expected behavior
Learner should have clear and immediate way to know exactly which resource are they currently looking at and interacting with, **without having to open the sidebar**. 

At a bare minimum the title of the resource should be inserted on the top of the main content as the main heading H1. 

<img width="1186" height="825" alt="Image" src="https://github.com/user-attachments/assets/36f0cc1f-6505-4ed0-bb9b-e5c329369e3e" />

I would also strongly suggest we include the Unit number somewhere, even if not as a semantic element. Maybe the easiest and most logical thing could be to precede it to the course title.

<img width="1186" height="501" alt="Image" src="https://github.com/user-attachments/assets/b545f298-a440-4755-88a2-f00796e1ae45" />

## User-facing consequences
Introducing this would stop learners wondering _Where the heck am I, and what am I doing...?_

## Steps to reproduce
…

## Context
 * Application version: asset from #14452 
 * Operating system: all
 * Browser: all

**Comment by marcellamaki** [comment]
We should use KBreadcrumb for Course > Unit > Resource title (but no links) rather than just the course name in the title, and the sidebar should be opened by default except on small screens

**Comment by rtibbles** [comment]
Fixed in #14557
